test/suite/test_compact09.py

```diff
@@ -78,8 +78,10 @@ def test_compact09(self):
         # Enable background compaction and exclude the two tables. Use run once to be able to
         # track the stats easily.
         exclude_list = f'["{self.uri_prefix}_0.wt", "{self.uri_prefix}_1.wt"]'
-        config = f'free_space_target=1MB,exclude={exclude_list},run_once=true'
-        self.turn_on_bg_compact(config)
+        config = f'background=true,free_space_target=1MB,exclude={exclude_list},run_once=true'
+        # Don't use the helper function as the server may go to sleep before we have the time to
```

Comment:
Could you please elaborate on what was happening here?

Reply:
With the helper function, we enable the server and wait for it to be enabled.
However, because it can be so quick, we have the following:

1. Python enabled the bg compaction server
2. The C code is exdcuted, bg compaction starts, analyses the files, `run_once` is on, goes back to sleep straight away
3. Python resumes, checks that bg compaction is running - too late, bg compaction is already done and back to sleep.

Reply:
Got it, thanks!